Build errors

I'm trying to build musly on Ubuntu 16.10. Got following error after `cmake ..`:
```
CMake Error at /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:148 (message):
  Could NOT find LibAV (missing: LIBAV_ROOT_DIR) (Required is at least
  version "0.8")
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:388 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindLibAV.cmake:218 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:34 (find_package)
```

If i run it like this `cmake .. -DLIBAV_ROOT_DIR=/usr` then I get compile error
```
/usr/src/musly/libmusly/decoders/libav.cpp: In member function ‘virtual std::vector<float> musly::decoders::libav::decodeto_22050hz_mono_float(const string&, float, float)’:
/usr/src/musly/libmusly/decoders/libav.cpp:182:5: error: ‘AVCodecParameters’ was not declared in this scope
     AVCodecParameters *decp = st->codecpar;
     ^~~~~~~~~~~~~~~~~
/usr/src/musly/libmusly/decoders/libav.cpp:182:24: error: ‘decp’ was not declared in this scope
     AVCodecParameters *decp = st->codecpar;
                        ^~~~
/usr/src/musly/libmusly/decoders/libav.cpp:182:35: error: ‘AVStream {aka struct AVStream}’ has no member named ‘codecpar’; did you mean ‘codec’?
     AVCodecParameters *decp = st->codecpar;
                                   ^~~~~~~~
/usr/src/musly/libmusly/decoders/libav.cpp:190:53: error: ‘avcodec_parameters_to_context’ was not declared in this scope
     avret = avcodec_parameters_to_context(decx, decp);                                                     ^
```
If i force this branch in code https://github.com/dominikschnitzer/musly/blob/master/libmusly/decoders/libav.cpp#L176, than it compiles successfully, but I don't know how to fix this properly for PR.

--- a/cmake/FindLibAV.cmake
+++ b/cmake/FindLibAV.cmake
@@ -24,6 +24,7 @@
 # Copyright (c) 2013 Sergiu Dotenco
 # Changed 2015 by Jan Schlüter: https://bitbucket.org/sergiu/libav-cmake/issue/2
 # Changed 2016 by Jan Schlüter: support both ffmpeg and libav
+# Changed 2017 by Sergey Smirnov: support architecture-specific include dirs
 #
 # Permission is hereby granted, free of charge, to any person obtaining a copy
 # of this software and associated documentation files (the "Software"), to deal
@@ -227,7 +228,8 @@ IF (NOT _LIBAV_CHECK_COMPONENTS)
  SET (_LIBAV_FPHSA_ADDITIONAL_ARGS HANDLE_COMPONENTS)
 ENDIF (NOT _LIBAV_CHECK_COMPONENTS)
 
-FIND_PACKAGE_HANDLE_STANDARD_ARGS (LibAV REQUIRED_VARS LIBAV_ROOT_DIR
-  LIBAV_INCLUDE_DIR ${_LIBAV_MISSING_LIBRARIES} VERSION_VAR LIBAV_VERSION
+FIND_PACKAGE_HANDLE_STANDARD_ARGS (LibAV
+  REQUIRED_VARS LIBAV_INCLUDE_DIR ${_LIBAV_MISSING_LIBRARIES}
+  VERSION_VAR LIBAV_VERSION
   ${_LIBAV_FPHSA_ADDITIONAL_ARGS})
 

